packages/go_router_builder/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 - Restricts `build` to versions less than 2.5.0.
 
+## 3.0.2
```

Comment:
I don't think it is possible to avoid the above in `Next` because that code is already landed and present. I think the path forward here is to include it as a bullet point under 3.0.2 and remove Next.